DS/linked_list.py

```
@@ -32,6 +33,32 @@ def __iter__(self):
             yield node
             node = node.next
 
+    def ll_length(self):
+        temp = self.head
+        count = 0
+        while (temp):
+            count += 1
+            temp = temp.next
+        return count
+
+
+    def insert(self, index, node):
+        """Insert at a given index"""
+        if index == 0 or self.head == None:
```

if I make a call as such, 
insert(12, None) // this should throw an error saying, such index does not exist in the LL.
however, in the current scenario, it'll just add a node at position 0.

---

I have added a new check for the same.